bundles/GraphicServiceOrder/Resources/config/services.yaml

```diff
@@ -13,8 +13,6 @@ services:
         resource: '../../Controller'
         tags: ['controller.service_arguments']
 
-    app.gs_upload_listener:
-        class: GraphicServiceOrder\EventListener\GsUploadListener
-        arguments: ["@doctrine.orm.entity_manager", "%kernel.project_dir%"]
-        tags:
-            - { name: kernel.event_listener, event: oneup_uploader.post_upload, method: onUpload }
+    GraphicServiceOrder\Service\FileUploader:
+        arguments:
+            $targetDirectory: '%env(GS_FILES_DIRECTORY)%'
```

Comment:
Missing newline.

Reply:
fixed

Reply:
@martinyde I think you should use `%env(resolve:GS_FILES_DIRECTORY)%` (cf. https://symfony.com/doc/current/configuration/env_var_processors.html).